packages/stylelint-plugin/test.less

```diff
@@ -1,14 +1,12 @@
 // This is a file you can use to play with the stylelint plugin
-
 @import "~@kaizen/design-tokens/less/color-vars";
-
 @import "~@kaizen/design-tokens/less/spacing-vars";
 
 .test {
-  color: rgba(@kz-var-color-cluny-600-rgb-params, 80%);
-  background-color: rgb(@kz-var-color-cluny-600-rgb-params);
+  color: rgba(@kz-var-color-blue-600-rgb-params, 80%);
+  background-color: rgb(@kz-var-color-blue-600-rgb-params);
 
-  border-color: rgba(@kz-var-color-wisteria-800-rgb-params, 70%);
+  border-color: rgba(@kz-var-color-purple-800-rgb-params, 70%);
```

Comment:
This whole file need a cleanup as they'll be stuck in an unfixable state (neither the old token names or the new ones)